pylmod/tests/test_gradebook.py

```diff
@@ -270,6 +480,20 @@ def test_get_gradebook_id(self):
             gradebook_id = test_gradebook.get_gradebook_id(self.GBUUID)
 
     @httpretty.activate
+    def test_get_options(self):
+        """Verify that we can get the options for a gradebook."""
+        self._register_get_gradebook()
+        self._register_get_options()
+        gradebook = GradeBook(self.CERT, self.URLBASE, self.GBUUID)
+        options = gradebook.get_options(gradebook_id=self.GRADEBOOK_ID)
+        self.assertEqual(
+            options['membershipQualifier'],
+            self.OPTIONS_BODY['data']['membershipQualifier']
+        )
+        # check for no data
```

Comment:
There isn't actually a check for not data here, so pull this or actually test it.